provider/terraform/resources/resource_cloudiot_registry.go

```diff
@@ -76,6 +76,7 @@ func resourceCloudIoTRegistry() *schema.Resource {
 			},
 			"mqtt_config": &schema.Schema{
 				Type:     schema.TypeMap,
+				Computed: true,
```

Comment:
So that Terraform doesn't see a diff! Since we have a complex object and the API returns a value every time, that object will always be present in state. We want to allow users to not explicitly set the block in their config, and so we need to make it `Computed` so that state can always be correct.